broker.go

```diff
@@ -16,22 +19,38 @@ import (
 )
 
 var (
-	discordToken string
-	finnhubToken string
+	discordToken  string
+	finnhubToken  string
+	messagePrefix string
+	test          bool
```

Comment:
Move these along with the "src" to be next to each other at the end of var() with a single newline above them,